Description: fix for jest-worker ≥ 27
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2021-09-26

--- a/rollup-plugin-terser.js
+++ b/rollup-plugin-terser.js
@@ -1,5 +1,5 @@
 const { codeFrameColumns } = require("@babel/code-frame");
-const Worker = require("jest-worker").default;
+const Worker = require("jest-worker").Worker;
 const serialize = require("serialize-javascript");
 
 function terser(userOptions = {}) {
